src/Microsoft.DotNet.Interactive/Connection/ProxyKernel.cs

```diff
@@ -74,8 +75,10 @@ internal override async Task HandleAsync(KernelCommand command, KernelInvocation
                     return;
             }
 
+            var kernelUri = KernelUri.Parse(command.TargetKernelName);
+            var remoteTargetKernelName = kernelUri.GetRemoteKernelName();
             var targetKernelName = command.TargetKernelName;
```

Comment:
Renaming this to `localTargetKernelName` might make it a bit more meaningful in context given these changes.